<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>E_InputHomeWid</class>
 <widget class="QWidget" name="E_InputHomeWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>721</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>电子可控负载一</string>
   </property>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_1" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_8" native="true">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>100</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_6" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_5" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_7" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>721</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>电子可控负载二</string>
   </property>
   <widget class="QWidget" name="widget_12" native="true">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_10" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_9" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_11" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_16" native="true">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>100</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_14" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_13" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_15" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>721</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>电子可控负载三</string>
   </property>
   <widget class="QWidget" name="widget_20" native="true">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_18" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_17" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_19" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_24" native="true">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>100</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_22" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_21" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_23" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
